src/coreclr/vm/stublink.cpp

```diff
@@ -1309,13 +1309,16 @@ bool StubLinker::EmitUnwindInfo(Stub* pStubRX, Stub* pStubRW, int globalsize, Lo
     //
 
     StubUnwindInfoHeader *pHeader = pStubRW->GetUnwindInfoHeader();
+    StubUnwindInfoHeader *pHeaderX = pStubRX->GetUnwindInfoHeader();
     _ASSERTE(IS_ALIGNED(pHeader, sizeof(void*)));
 
     BYTE *pbBaseAddress = pbRegionBaseAddress;
```

Comment:
Thank you for fixing this. I am actually surprised this is not used without the interpreter enabled.
A nit -can you please change the naming so that we have RX and RW suffixes for all pointers that we have in the dual form? So:
```
pHeader -> pHeaderRW
pHeaderX -> pHeaderRX
pbBaseAddress -> pbBaseAddressRX
pbBaseAddressW -> pbBaseAddressRW
```